config/load_test.go

```diff
@@ -23,6 +23,17 @@ func TestLogLevel(t *testing.T) {
 }
 
 func TestServerConfig(t *testing.T) {
+	// default value test
+	serverConfigLoader()
+	assert.Equal(t, true, Server.DedupEnabled)
+
+	// override value test
+	os.Setenv("SERVER_DEDUP_ENABLED", "false")
```

Comment:
Could we name it more explainable. for eg.
SERVER_BATCH_DEDUP_IN_CONNECTION_ENABLED

Reply:
done.